src/algebra/group/conj.lean

```diff
@@ -44,6 +45,18 @@ begin
   repeat {rw mul_assoc},
 end
 
+/-- `b * a * b⁻¹` as a bundled equiv. -/
+@[simps apply]
+def conj_hom (b : α) : α ≃* α :=
```

Comment:
I think we use `_equiv` as suffix for equivs, and `_hom` for "mere" homomorphisms. At least that's what I've been doing...

Otherwise LGTM

Reply:
```suggestion
def conj_equiv (b : α) : α ≃* α :=
```